packages/orbit-components/transforms/tokens-v16.mjs

```diff
@@ -0,0 +1,75 @@
+import { readFile, writeFile, readdir, stat } from "fs/promises";
```

Comment:
A simple search and replace within a given folder.

We might want to improve/extend this script so that it's possible to run it in a specific folder instead (over navigating to the folder and then running the script).